README.md

```diff
@@ -86,7 +86,8 @@ Edit your `package.json`:
         }
       },
       "dotnet": {
-        "namespace": "Acme.Hello"
+        "namespace": "Acme.Hello",
+        "packageId": "Acme.Hello"
```

Comment:
Set a different packageId here to make it clear where this value is going

Reply:
✔️